Makefile

```diff
@@ -140,10 +142,18 @@ prettier: | $(NODELIB)
 check-prettier: | $(NODELIB)
 	npx --prefix $(NODEPREFIX) -y prettier --check .
 
+# TODO: BASEURL should be calculated dynamically for CI/PR.
+e2e: E2EDIR=e2e
```

Comment:
Is `E2EDIR` a variable just to reduce repetition? Or is there a use case for running `make e2e E2EDIR=foo` ? If the former I think I'd get rid of the variable and hard-code it:

	npm --prefix e2e ci
	npx --prefix e2e playwright install --with-deps chromium
	npx --prefix e2e playwright test --config e2e

But up to you.